subscribers.go

```diff
@@ -127,6 +143,7 @@ func (s *subscriberService) Get(ctx context.Context, options *GetSubscriberOptio
 	return root, res, nil
 }
 
+// Deprecated: use Upsert instead
```

Comment:
I would also mark it as deprecated in the interface—if someone is using an interface, it won't be visible 😉  

Maybe it would be good to add some context about why it's deprecated.

Reply:
added link back to issue  (https://github.com/mailerlite/mailerlite-go/issues/17) and marked interface as Deprecated 👍